applications/mne_analyze/libs/anShared/Model/bemdatamodel.cpp

```diff
@@ -113,8 +113,9 @@ void BemDataModel::initBemData(QIODevice& qIODevice)
 
 //=============================================================================================================
 
-QVariant BemDataModel::data(const QModelIndex &index,
-                            int role) const
+QVariant BemDataModel::data(const QModelIndex &,//index,
+                            int )//role)
+                            const
```

Comment:
Is there a specific reason why the variables were removed?

Reply:
This and analogous edits are really in the interest of making the warnings at least not disturb and distract while we are developing. There are aprox. 150 now. Mainly in areas of the codebase where we're not working at the moment. So if we are able to achieve zero warnings, then we would really take matters more seriously whenever there are in the areas of the code where we are working. That is my -broken?- train of though. 

Regarding this precise edit. Q_UNUSED(VAR) generates another warning which only stops when you remove the var name in the method definition.